pamelalozano/src/main/webapp/script.js

```diff
@@ -18,23 +18,90 @@ document.getElementById("projects").addEventListener("click", function (event) {
         event.target.closest('.project').classList.toggle("open");
 });
 
+let cursor = "";
+
 /** Loads existing Comments */
 async function loadComments() {
   let response = await fetch('/data');
-  let comments = await response.json();
+  let responseJson = await response.json();
+  let comments = responseJson.comments;
+  cursor = responseJson.cursor;
+
   if(comments.length == 0) {
-      document.getElementById("comments-section").innerHTML="No comments";
+      document.getElementById("comments-list").innerHTML="No comments";
   } 
   else {
-    let commentSection = document.getElementById("comments-section");
+    let commentSection = document.getElementById("comments-list");
+    
+    //The newest comments at the top
+    comments.forEach(comment => {
+        console.log(comment);
+        let newCard = createCard(comment);
+        commentSection.append(newCard);
+    });
+
+  };
+}
+
+async function nextPage(){
```

Comment:
Since these are so similar, you can create a helper function that each of these functions can call to avoid repeating code.

Reply:
Yeah I actually mentioned this in the PR description, it's done ! :)